commit/proximals.pyx

```diff
@@ -53,9 +53,9 @@ cpdef projection_onto_l2_ball(np.ndarray[np.float64_t] x, double lam, int compar
         np.ndarray[np.float64_t] v
         size_t i
     v = x.copy()
-    xn = sqrt(sum(v[compartment_start:compartment_size]**2))
+    xn = sqrt(sum(v[compartment_start:(compartment_start+compartment_size)]**2))
```

Comment:
The notation is not coherent with the other corrections. I suggest to cut the parenthesis.